meta-resin-common/classes/kernel-resin.bbclass

```diff
@@ -121,6 +121,7 @@ RESIN_CONFIGS[docker] ?= " \
     CONFIG_EXT4_FS_SECURITY=y \
     CONFIG_AUFS_FS=y \
     CONFIG_AUFS_XATTR=y \
+    CONFIG_IP_NF_TARGET_REJECT=y \
```

Comment:
This was actually added now at some point (albeit as a module): https://github.com/resin-os/meta-resin/blob/master/meta-resin-common/classes/kernel-resin.bbclass#L119 - it was needed for the supervisor's iptables rules to work